fastestimator/network/network.py

```diff
@@ -85,27 +87,29 @@ def _get_signature_epoch(self, mode):
     def load_epoch(self, epoch, mode):
         ops = self.op_schedule[mode].get_current_value(epoch)
         model_list = self.model_schedule[mode].get_current_value(epoch)
-        loss_list = []
+        losses_epoch = []
         for model in model_list:
-            if model.loss_name not in loss_list:
-                loss_list.append(model.loss_name)
-        self.loss_list = loss_list
-        return ops, model_list, loss_list
+            if model.loss_name not in losses_epoch:
```

Comment:
we might want to make losses_epoch a set rather than list to speed up the 'not in' check

Reply:
I think the computational complexity for member check is the same.   Also, set is not iterable, which will create problems when we try to iterate through them.

Reply:
sets are iterable. also, unless python did sets totally wrong, membership check in set is O(1) vs in list O(n)